crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -716,7 +716,7 @@ impl Binary {
         Binary::new(operator, lhs, rhs)
     }
 
-    fn evaluate<F>(
+    pub fn evaluate<F>(
```

Comment:
I believe you do not need to make it pub

Reply:
This was an old change I forgot to remove, done